examples/with-react-esi/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "with-react-esi",
+  "author": "Kévin Dunglas <dunglas@gmail.com>",
+  "main": "dist/server.js",
+  "dependencies": {
+    "express": "^4.16.4",
+    "next": "^7.0.2",
+    "prettier": "^1.15.3",
+    "react": "^16.7.0",
+    "react-dom": "^16.7.0",
+    "react-esi": "^0.1"
+  },
+  "scripts": {
+    "dev": "nodemon --exec babel-node src/server.js localhost 3000",
+    "build": "babel src -d dist && next build dist",
```

Comment:
This is not needed: `babel src -d dist &&`

Reply:
It's weird to run `babel` over the `src` directory 🤔 You'd be tripple parsing.

Reply:
The problem is that React ESI needs to be able to serve the components (`src/components`)  directly. Is it possible to reuse the ones compiled by Next?

Reply:
Oh I see, that makes it indeed harder, all components are put into a single bundle per page and are not available to be required outside of the bundle, so that is an issue.